ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -165,6 +165,11 @@ public void updateExtraData(ReactEditText view, Object extraData) {
     }
   }
 
+  @ReactProp(name = ViewProps.COLOR, defaultInt = 0)
+  public void setColor(ReactEditText view, @Nullable Integer color) {
```

Comment:
You should probably say that this has customType="color" as well.

I'm not sure if defaultInt will keep null from actually showing up.  If null does show up or defaultInt you probably need to set it back to whatever it was before.